### PR TITLE
Support HTTP headers

### PR DESCRIPTION
**Pull request by alexkwolfe**
HTTP request headers can now be mapped.

screen shot 2016-02-12 at 3 07 15 pm

### CODE REVIEW

**Comment by alexkwolfe** [comment]
Yet ANOTHER feature that LCC doesn't have. En garde!

**Comment by alexkwolfe** [comment]
Er. Wait. I guess it does in the Custom HTTP POST. :disappointed:

File: src/variables.coffee [hunk 1]
```diff
@@ -4,4 +4,5 @@ module.exports = [
   { name: 'outcome_on_match', description: 'The outcome when the term is found - "success" or "failure" (default: success)', type: 'string', required: true }
   { name: 'reason_path', description: 'The dot-notation path, XPath location, or regular expression with a single capture group, used to find the failure reason', type: 'string', required: false }
   { name: 'default_reason', description: 'Failure reason when no reason can be found per the optional Reason Path setting', type: 'string', required: false }
+  { name: 'header.*', description: 'HTTP header to send in the request', type: 'wildcard', required: false }
```
Comment (cgrayson) [inline src/variables.coffee]: Maybe list the 3 disallowed ones here?

**Comment by alexkwolfe** [comment]
Touche!

**Comment by cgrayson** [comment]
Please don't touché yourself on company time.

File: src/variables.coffee (hunk 1 above)
Comment (alexkwolfe) [inline src/variables.coffee]: If you think so, but I honestly don't think anyone will try to map those. I even considered just silently overwriting them.

**Comment by cgrayson** [comment]
Otherwise LGTM.

(Maybe not as good as _this_, but "good"):
screen shot 2016-02-12 at 3 20 22 pm

**Comment by alexkwolfe** [comment]
That is actually pretty nice.

**Comment by alexkwolfe** [comment]
I'll watch for those header errors. If we see some, let's revisit the description. It would be nice not to make it too long.

**Pull request merged**